modules/solvers/voltage-divider.stanza

```diff
@@ -25,23 +25,27 @@ public defstruct VoltageDividerSolution :
   vo: Toleranced
 
 public defn voltage-divider-solver (v-in: Toleranced,
-                             v-out: Toleranced,
-                             current: Double) -> VoltageDividerSolution|False:
+                                    v-out: Toleranced,
+                                    current: Double) -> VoltageDividerSolution|False:
   ; Look for highest acceptable tolerance in EIA standard values: [20% 10% 5% 2% 1% 0.5% 0.25% 0.1%]
   val current-tolerance = 0.1
-  val goal-r-hi = (typ-value(v-in) - typ-value(v-out)) / current
-  val goal-r-lo = typ-value(v-out) / current
+  val goal-r-hi = (v-in - v-out) / current
+  val goal-r-lo = v-out / current
   label<VoltageDividerSolution|False> return :
-    for tol in std-tols do :
+    for std-tol in std-tols do :
       ; std-tols are in % instead of being unitless
-      val tolerance = tol / 100.
+      val tolerance = std-tol / 100.
+      val lo-tol = tol(0.0, tolerance)
+      val hi-tol = tol(0.0, tolerance)
```

Comment:
This formulation allows negative tolerances which should be a no-no.

Reply:
This commit should prohibit negative tolerances.
`std-tols` range up to 20.0.
`tolerance` will be at most `20.0 / 100.0 = 0.2`.
`tol(1.0, tolerance)` has min-value of at least 0.8